### PR TITLE
Updated sidecar

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
Updated sidecar
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/491 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

### CODE REVIEW

File: operatorconfig/driverconfig/common/k8s-1.22-values.yaml [hunk 1]
```diff
@@ -2,26 +2,26 @@
 images:
   # "images.attacher" defines the container images used for the csi attacher
   # container.
```
Comment (rensyct) [inline operatorconfig/driverconfig/common/k8s-1.22-values.yaml:4]: Please revert the changes in this file as we are not supporting K8s v1.22

**Review by mbasha-dell — APPROVED** [review]

File: operatorconfig/driverconfig/common/k8s-1.22-values.yaml (hunk 1 above)
Comment (karthikk92) [reply operatorconfig/driverconfig/common/k8s-1.22-values.yaml]: taken care

**Review by panigs7 — APPROVED** [review]

**Review by rensyct — APPROVED** [review]
